doc/api/buffer.md

```diff
@@ -1588,6 +1588,14 @@ console.log(buf.readIntBE(0, 6).toString(16));
 
 // Throws an exception: RangeError: Index out of range
 console.log(buf.readIntBE(1, 6).toString(16));
+
+// Throws an exception: "byteLength" argument must be a number
+console.log(buf.readIntBE(1).toString(16));
```

Comment:
Might as well adding `TypeError: ` and `RangeError: ` as the previous example, just to be a little bit more consistent.